### PR TITLE
:arrow_up: Updates pages for dev_console rollout

### PR DESCRIPTION
**Pull request by jmelberg-okta**
## Description:
- Updates pages referencing Admin pages with new Dev Console.

### Resolves:

* [OKTA-135690](https://oktainc.atlassian.net/browse/OKTA-135690)

### CODE REVIEW

File: _source/_code/javascript/okta_sign-in_widget.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ python3 -m http.server
 
 This step is necessary for Okta to accept authentication requests from an application through the Sign-In Widget.
 
-You can enable your Sign-in Widget page as a "Trusted Origin" in Okta, which enables Cross-Origin Resource Sharing (CORS). To do this, follow the steps found under the "Trusted Origins tab" section in our [API Security help page](https://help.okta.com/en/prev/Content/Topics/Security/API.htm). Configure your Origin using the same base URL as the web server you are using to host the HTML for the Okta Sign-In Widget. For example, if you're serving the page using the Python SimpleHTTPServer, the origin would be `http://0.0.0.0:8000`.
+You can enable your Sign-in Widget page as a "Trusted Origin" in Okta, which enables Cross-Origin Resource Sharing (CORS). To do this, follow the steps found under the **API** > **Trusted Origins**. Configure your Origin using the same base URL as the web server you are using to host the HTML for the Okta Sign-In Widget. For example, if you're serving the page using the Python SimpleHTTPServer, the origin would be `http://0.0.0.0:8000`.
```
Comment (mystiberry-okta) [inline _source/_code/javascript/okta_sign-in_widget.md]: no "the" in front of **API** > **Trusted Origins**

File: _source/_docs/api/getting_started/enabling_cors.md [hunk 2]
```diff
@@ -30,21 +30,21 @@ Not all browsers supports CORS.  The following table describes which browsers su
 
 ## Granting Cross-Origin Access to Websites
 
-You can enable CORS for websites that need cross-origin requests to the Okta API on the Okta Admin Dashboard. Select **Security** and then, **API**. Select the CORS tab and select **Edit** to see the screen shown below.
+You can enable CORS for websites that need cross-origin requests to the Okta API on the Okta Developer Dashboard. Select **API** and then, **Trusted Origins** to see the screen shown below.
```
Comment (mystiberry-okta) [inline _source/_docs/api/getting_started/enabling_cors.md]: No comma after "and then" . Should probably be consistent and use >

Comment (mystiberry-okta) [inline _source/_docs/api/getting_started/enabling_cors.md]: Let's make sure this approach (not saying you're in the Okta UI, not starting from either landing page) is okay with customers. I'm concerned that it'll be too out of context, but not sure enough to say 'don't do it.'

File: _source/_docs/how-to/set-up-auth-server.md [hunk 3]
```diff
@@ -27,11 +26,9 @@ How do you know if you need to use Okta's Authorization Server instead of the au
 built in to your Okta app?
 
 * You need to protect non-Okta resources.
-* You need different authorization policies depending on whether the person is an employee, partner, or end user, or
-other similar specializations.
+* You need different authorization policies depending on whether the person is an employee, partner, or end user, or other similar specializations.
 
-> Note: If your employees, partners, and users can all use the same authentication policies for single sign-on,
-try [Okta's built in authorization service](https://support.okta.com/help/articles/Knowledge_Article/Single-Sign-On-Knowledge-Hub).
+> Note: If your employees, partners, and users can all use the same authentication policies for single sign-on, try [Okta's built in authorization service](https://support.okta.com/help/articles/Knowledge_Article/Single-Sign-On-Knowledge-Hub).
```
Comment (mystiberry-okta) [inline _source/_docs/how-to/set-up-auth-server.md]: It's not at all clear what we mean by built-in auth service.  Can we make it "built-in" and also use the name SSO or something?

Comment (jmelberg-okta) [inline _source/_docs/how-to/set-up-auth-server.md]: Great catch. It isn't clear what "Authorization Service" is in this context.

**Pull request merged**